File: backend/cloud-publishing-rest/employee-rest-service/src/main/java/com/ikuzmin/employee/rest/EmployeeRestServiceWebApp.java
package com.ikuzmin.employee.rest;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.boot.autoconfigure.domain.EntityScan;
import org.springframework.boot.web.servlet.support.SpringBootServletInitializer;
import com.ikuzmin.cloud.publishing.model.entitys.Employee;


/**
 *
 * @author Igor Kuzmin
 */
@SpringBootApplication
@EntityScan(basePackageClasses = Employee.class)
public class EmployeeRestServiceWebApp extends SpringBootServletInitializer {

  public static void main(String[] args) {
    SpringApplication.run(EmployeeRestServiceWebApp.class, args);
  }

}

File: backend/cloud-publishing-rest/employee-rest-service/src/main/kotlin/com/ikuzmin/employee/rest/EmployeeRestServiceWebApp.kt
package com.ikuzmin.employee.rest

import com.ikuzmin.cloud.publishing.model.entities.Profile
import com.ikuzmin.common.rest.clients.RestClientsContext
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.autoconfigure.domain.EntityScan
import org.springframework.boot.runApplication
import org.springframework.context.annotation.Import
import org.springframework.scheduling.annotation.EnableAsync
import org.springframework.scheduling.annotation.EnableScheduling


@SpringBootApplication
@EntityScan(basePackageClasses = [Profile::class])
@Import(RestClientsContext::class)
@EnableScheduling
@EnableAsync
class EmployeeRestServiceWebApp;

fun main(args: Array<String>) {
    runApplication<EmployeeRestServiceWebApp>(*args)
}